vmm/src/device_manager.rs

```diff
@@ -802,6 +802,58 @@ impl AccessPlatform for SevSnpPageAccessProxy {
     }
 }
 
+/// Helpers for advisory file locking following best-practices described in
+/// <https://apenwarr.ca/log/20101213>.
+#[cfg(target_family = "unix")]
```

Comment:
When would target_family not be unix?  We do Linux syscalls all over the codebase and have zero target_family checks.

Reply:
Oh, you are right. That is not necessary for CHV.